email_notification/helpers.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+from permissions.helpers import has_unit_permission
+from reservations.models import Reservation
+from spaces.models import Unit
+from users.models import ReservationNotification, User
+
+
+def get_notification_recipients(
```

Comment:
Would  `get_staff_notification_recipients` be more descriptive name?

Reply:
It would indeed! Renamed 👍